ember/app/ui/components/identity-members/component.js

```diff
@@ -21,63 +20,32 @@ export default class IdentityMembersComponent extends Component {
     return this.pageNumber < this.totalPages;
   }
 
-  @restartableTask
-  *fetchMembers() {
-    const membershipResponse = yield this.store.query(
-      "membership",
+  get locale() {
+    return this.intl.locale[0];
+  }
+
+  _members = trackedFunction(this, async () => {
```

Comment:
Nitpick: I would just rename this to `membersResource` because that's what it is.

Reply:
renamed